backend/src/services/services.ts

```diff
@@ -1,10 +1,20 @@
 import { USER_PASSWORD_SALT_ROUNDS } from '~/common/constants/user.constants';
 import { ENV } from '~/common/enums/enums';
-import { user as userRepository } from '~/data/repositories/repositories';
+import {
+  user as userRepository,
+  groups as groupsRepository,
+  permission as permissionRepository,
+  groupsToPermissions as groupsToPermissionsRepository,
+  usersToGroups as usersToGroupsRepository,
+} from '~/data/repositories/repositories';
 import { Auth } from './auth/auth.service';
 import { Encrypt } from './encrypt/encrypt.service';
 import { Token } from './token/token.service';
 import { User } from './user/user.service';
+import { Groups } from './groups/groups.service';
```

Comment:
```suggestion
import { Group } from './group/group.service';
```
should be in singular state